torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -1853,36 +1853,35 @@ void TensorExprKernel::compile() {
 
 TensorExprKernel::TensorExprKernel(const std::shared_ptr<Graph>& subgraph)
     : graph_(subgraph), code_(subgraph, "") {
-  if (!fallbackAllowed()) {
+
+  allow_fallback_ = fallbackAllowed();
+  if (!allow_fallback_) {
     compile();
     return;
   }
 
+  use_fallback_ = fallbackEnforced();
```

Comment:
nitpick: maybe renaming `use_fallback_` to `failed_comp_or_required_fallback_` makes it a bit more obvious in which cases we are using the fallback?